src/components/BibleDropDown/BibleDropDown.md

```diff
@@ -21,26 +23,55 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const [bookName, setBookName] = useState([]);
+const [bookList, setBookList] = React.useState([]);
+const [projectBooks, setProjectBooks] = useState(['gen', 'exo']);
+const [sourceBooks, setSourceBooks] = useState();
 const classes = useStyles();
 
+const loadBibleBooks = () => {
+  API.get('bibles/en_KJV_1_bible/books')
```

Comment:
en_KJV_bible should ideally not be hard coded as it may or may not be uploaded, instead the source list dropdown should be used and the user allowed to select a valid source fromt there. We can leave it for now we can take it up when redoing the UI